examples/graph_visual_with_falkordb.py

```
@@ -0,0 +1,266 @@
+import os
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)